packages/core/strapi/src/node/vite/config.ts

```diff
@@ -85,6 +85,7 @@ const resolveDevelopmentConfig = async (ctx: BuildContext): Promise<InlineConfig
       },
     },
     server: {
+      cors: false,
```

Comment:
Is it needed for this PR? Does serving from different paths causes cors issues?

Reply:
It was causing some issues in the past, and I figure it doesn't make sense to have that conflict with our own cors middleware basically. I don't see where there would be an issue considering it's only in dev mode